app/dashboard/views.py

```diff
@@ -3263,6 +3264,65 @@ def verify_user_google(request):
 
     return redirect('profile_by_tab', 'trust')
 
+def connect_facebook():
+    import urllib.parse
+
+    facebook = OAuth2Session (
+        settings.FACEBOOK_CLIENT_ID,
+        redirect_uri=urllib.parse.urljoin(settings.BASE_URL, reverse(verify_user_facebook)),
+    )
+
+    return facebook_compliance_fix(facebook)
+
+@login_required
+@require_POST
+def request_verify_facebook(request, handle):
+    is_logged_in_user = request.user.is_authenticated and request.user.username.lower() == handle.lower()
+    if not is_logged_in_user:
+        return JsonResponse({
+            'ok': False,
+            'msg': f'Request must be for the logged in user',
+        })
+
+    profile = profile_helper(handle, True)
+    if profile.is_facebook_verified:
```

Comment:
Is this checking for some kind of edge case where the button is still active but the profile is already facebook verified? like multiple browser tabs open or something like that?

Reply:
Yes exactly, any case that facebook status already verified but the profile trust page hasn't been updated.